<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dot11 Analyzer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>ESSID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STA</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Signal</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Beacons</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ch</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ecryption</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cipher</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Auth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>BSSID</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionInterface"/>
   </widget>
   <addaction name="menuCapture"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionClear"/>
   <addaction name="actionInterface"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="icon_image/icon_image.qrc">
     <normaloff>:/icon_image/start_icon.png</normaloff>:/icon_image/start_icon.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icon_image/icon_image.qrc">
     <normaloff>:/icon_image/stop_icon.png</normaloff>:/icon_image/stop_icon.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icon_image/icon_image.qrc">
     <normaloff>:/icon_image/eraser_icon.png</normaloff>:/icon_image/eraser_icon.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionInterface">
   <property name="icon">
    <iconset resource="icon_image/icon_image.qrc">
     <normaloff>:/icon_image/settings_icon.png</normaloff>:/icon_image/settings_icon.png</iconset>
   </property>
   <property name="text">
    <string>Interface</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon_image/icon_image.qrc"/>
 </resources>
 <connections/>
</ui>
